gen/teams_provisioning

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Perun::Agent;
+use JSON::XS;
+use Data::Dumper;
+
+our $SERVICE_NAME = "teams_provisioning";
+our $PROTOCOL_VERSION = "3.0.2";
+my $SCRIPT_VERSION = "3.0.2";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+#Constants
+our $A_RES_ALIAS;             *A_RES_ALIAS =         \'urn:perun:resource:attribute-def:def:o365ResourceAlias';
+our $A_USER_LOGIN;            *A_USER_LOGIN=             \'urn:perun:user:attribute-def:def:login-namespace:mu';
+
+my @teams = ();
+foreach my $resourceId ($data->getResourceIds()) {
+		my $resAlias = data->getResourceAttributeValue(resource=>$resourceId, attrName=>$A_RES_ALIAS);
+        print $resAlias . "\n";
+        my $team->{"identity"} = $resAlias;
+        my @members = ();
+        foreach my $memberId ($data->getMemberIdsForResource($resourceId)) {
+			my $userLogin = $data->getMemberAttributeValue(member=> $memberId, attrName=>$A_USER_LOGIN);
+            print $userLogin . "\n";
+            push @members, $userLogin;
+        };
+        $team->{"members"} = \@members;
```

Comment:
I suggest to sort members in the list in order to get comparable outputs between script versions/runs. You can add following above this line: `@members = sort(@members);`.